kerlescan/view_helpers.py

```diff
@@ -101,6 +112,25 @@ def ensure_account_number(request, logger):
         raise HTTPError(HTTPStatus.BAD_REQUEST, message="identity not found on request")
 
 
+def ensure_org_id(request, logger):
+    if _is_mgmt_url(request.path) or check_request_from_turnpike(
+        request=request, logger=logger
+    ):  # TODO: pass in app_name for openapi url check
+        return  # allow request
+
+    auth_key = get_key_from_headers(request.headers)
+    if auth_key:
+        identity = json.loads(base64.b64decode(auth_key))["identity"]
+        if "org_id" not in identity:
+            logger.debug("org id not found on identity token %s" % auth_key)
+            raise HTTPError(
+                HTTPStatus.BAD_REQUEST,
+                message="org id not found on identity token",
+            )
+    else:
+        raise HTTPError(HTTPStatus.BAD_REQUEST, message="identity not found on request")
```

Comment:
You are right, we write `identity` into logs for `auth_key` in `headers` as that header is present only with proper identity of the user. Currently we have the same with the `ensure_account_number` method just above. I just copied it :-)

We can change it so it writes more technical message into logs using `auth_key` instead of `identity` but I would do it in different PR.

Reply:
Ok, fair point.  If it will always be present when auth_key is, I'm ok with moving ahead as is.  I would not worry about additional PRs at the moment with Drift in maintenance mode unless it came up as a concern elsewhere and prioritized.